.github/workflows/commons-ci.yml

```diff
@@ -0,0 +1,63 @@
+name: "Graph Commons CI"
```

Comment:
prefer "HugeGraph-Commons CI", and please also improve the names in other ci.yml files?

Reply:
@imbajin which perfer?

Reply:
<img width="628" alt="image" src="https://github.com/user-attachments/assets/f4495ffb-ad32-4a0a-849d-1ad19fce3840">

prefer simple one

Reply:
> I prefer to use the component names directly to express the ci-job meaning precisely, so that newcomers who don't know the background can understand it quickly.

We could ask some newcomers for their opinions:
@haohao0103 @zhenyuT @ChenZiHong-Gavin @vichayturen @returnToInnocence @jasinliu @ZeeJJ123 @AnkiPu @HJ-Young @Thespica

Which CI name do u prefer to choose: ( `Graph-XX-CI` or `HugeGraoh-XX-CI`
<img width="890" alt="image" src="https://github.com/user-attachments/assets/44e19e33-2150-4f7b-bf47-bb072e4ed0cb">, u could choose to support it with emoji 🚀 

Or choose to unify to ↓:
```markdown
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / pd (pull_request)
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / store (pull_request)
- HugeGraph-PD & HugeGraph-Store & HugeGraph-Hstore CI / hstore (pull_request)
- HugeGraph-Server CI / build-server (rocksdb, 11) (pull_request)
- HugeGraph-Server CI / build-server (mysql, 11) (pull_request)
- HugeGraph-Server CI / build-server (hbase, 11) (pull_request)
- ....
- HugeGraph-Server CI / xxx-...
```

u could choose to support it with emoji  👀

Reply:
For the second option, I feel the CI name is too long. I think a compromise would be to change Graph to HugeGraph while keeping the rest of the format unchanged.

Reply:
Choose full format based on voting.